prime-router/metadata/fhir_mapping/hl7/datatypes/NDL/PractitionerRole.yml

```diff
@@ -0,0 +1,120 @@
+resourceType: PractitionerRole
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+practitioner:
+  valueOf: datatypes/CNN/Practitioner
+  expressionType: reference
+  specs: NDL.1
+
+period:
+  expressionType: nested
+  expressionsMap:
+    start:
+      valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
+      expressionType: JEXL
+      vars:
+        dateTimeIn: NDL.2
+      type: STRING
+    end:
+      valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
+      expressionType: JEXL
+      vars:
+        dateTimeIn: NDL.3
+      type: STRING
+
+location:
+  generateList: true
+  expressionType: nested
+  expressions:
+    - valueOf: datatypes/NDL/Location
+      condition: $ndl7 NOT_NULL
+      expressionType: reference
+      specs: NDL.7
+      vars:
+        ndlLocationStatus: NDL.8
+        ndl7: NDL.7
+      constants:
+        mode: "instance"
+        physicalTypeCode: "si"
+    - valueOf: datatypes/NDL/Location
+      condition: $ndl10 NOT_NULL
+      expressionType: reference
+      specs: NDL.10
+      vars:
+        refLocation: $BASE_VALUE, GeneralUtils.extractAttribute(refLocation,"$.identifier[0].value","String")
```

Comment:
This currently does not work as expected and there will be a follow up ticket to handle it as it's not strictly necessary to accomplish this mapping